store/sqlstore/team_store.go

```diff
@@ -454,104 +498,131 @@ func (s SqlTeamStore) GetAllPage(offset int, limit int) ([]*model.Team, *model.A
 
 func (s SqlTeamStore) GetTeamsByUserId(userId string) ([]*model.Team, *model.AppError) {
 	var teams []*model.Team
-	if _, err := s.GetReplica().Select(&teams, "SELECT Teams.* FROM Teams, TeamMembers WHERE TeamMembers.TeamId = Teams.Id AND TeamMembers.UserId = :UserId AND TeamMembers.DeleteAt = 0 AND Teams.DeleteAt = 0", map[string]interface{}{"UserId": userId}); err != nil {
+	query, args, err := s.getQueryBuilder().
+		Select("Teams.*").
+		From("Teams").
```

Comment:
Maybe we could leverage `s.teamsQuery` here.

Reply:
Done.